spark/v3.2/build.gradle

```diff
@@ -132,6 +136,8 @@ project(":iceberg-spark:iceberg-spark-3.2-extensions") {
     compileOnly("org.apache.spark:spark-hive_2.12:${sparkVersion}") {
       exclude group: 'org.apache.avro', module: 'avro'
       exclude group: 'org.apache.arrow'
+      // to make sure io.netty.buffer only comes from project(':iceberg-arrow')
+      exclude group: 'io.netty', module: 'netty-buffer'
```

Comment:
We don't actually need this line make jmh work with the new change in iceberg-arrow, but I still keep them to make sure netty buffer only comes from iceberg-arrow. I'm open to any suggestion.